src/ripple/app/ledger/InboundLedger.h

```diff
@@ -160,6 +162,8 @@ class InboundLedger
 
     std::set <uint256> mRecentNodes;
 
+    SHAMapAddNode      mStats;
```

Comment:
I think that having two different naming conventions in the same class is worse than using a naming convention that isn't the best one.